---a/net/minecraft/util/ChatComponentTranslationFormatException.java
+++b/net/minecraft/util/ChatComponentTranslationFormatException.java
@@ -1,22 +1,19 @@
 package net.minecraft.util;
 
-public class ChatComponentTranslationFormatException extends IllegalArgumentException
-{
+public class ChatComponentTranslationFormatException extends IllegalArgumentException {
+
     private static final String __OBFID = "CL_00001271";
 
-    public ChatComponentTranslationFormatException(ChatComponentTranslation component, String message)
-    {
-        super(String.format("Error parsing: %s: %s", new Object[] {component, message}));
+    public ChatComponentTranslationFormatException(ChatComponentTranslation component, String message) {
+        super(String.format("Error parsing: %s: %s", new Object[]{component, message}));
     }
 
-    public ChatComponentTranslationFormatException(ChatComponentTranslation component, int index)
-    {
-        super(String.format("Invalid index %d requested for %s", new Object[] {Integer.valueOf(index), component}));
+    public ChatComponentTranslationFormatException(ChatComponentTranslation component, int index) {
+        super(String.format("Invalid index %d requested for %s", new Object[]{Integer.valueOf(index), component}));
     }
 
-    public ChatComponentTranslationFormatException(ChatComponentTranslation component, Throwable cause)
-    {
-        super(String.format("Error while parsing: %s", new Object[] {component}), cause);
+    public ChatComponentTranslationFormatException(ChatComponentTranslation component, Throwable cause) {
+        super(String.format("Error while parsing: %s", new Object[]{component}), cause);
     }
-}
+}
